### PR TITLE
Improve python to instruct user how to fix

### PR DESCRIPTION
**Pull request by gabe-terrell**
Removes commands that are pulled over from starters that will not map
to a user’s existing project. Additionally create a non-intrusive README that
explains to the user what files must be changed to complete the enablement process.

### CODE REVIEW

**Comment by gabe-terrell** [comment]
I'll update the conflicts, but I'm a bit stumped by Codacy's complaint. It's saying:
```
 Instead of `pip install <package>` use `pip install <package>==<version>`
```
but we're just trying to update pip to its latest version:
```
RUN pip install --upgrade pip
```
As far as I am aware, we don't want to to specify a package version, we just want to update it to its latest version, and that is done via the `--upgrade` flag. Any suggestions on how to fix this?

**Comment by gibfahn** [comment]
>As far as I am aware, we don't want to to specify a package version, we just want to update it to its latest version, and that is done via the --upgrade flag. Any suggestions on how to fix this?

That does sound like an issue with Codacy. Looks like someone with the right powers could ignore this particular issue in this file, hopefully @cvignola has those powers.

File: generators/dockertools/index.js [hunk 1]
```diff
@@ -288,12 +299,12 @@ module.exports = class extends Generator {
 			dockerFileTools: 'Dockerfile-tools',
 			imageNameRun: `${applicationName.toLowerCase()}-flask-run`,
 			imageNameTools: `${applicationName.toLowerCase()}-flask-tools`,
-			buildCmdRun: 'python -m compileall server test',
-			testCmd: 'python -m unittest tests.app_tests.ServerTestCase',
-			buildCmdDebug: 'python -m compileall server test',
+			buildCmdRun: 'python -m compileall .',
```
Comment (Aerex) [inline generators/dockertools/index.js]: Good idea but we need to add a flag under [Starter-Bluemix-Basic](https://github.ibm.com/arf/starter-bluemix-basic) to show that we are getting a project from someplace other than a generated starter. In addition, we need to add some a flag under [generator-readme](https://github.ibm.com/arf/generator-readme) to document to the user if he uses `bx dev enable` you will need to manually modify the `cli-config` to fit their usages.

File: generators/dockertools/index.js [hunk 2]
```diff
@@ -288,12 +299,12 @@ module.exports = class extends Generator {
...
+			testCmd: 'echo No test command specified in cli-config',
+			buildCmdDebug: 'python -m compileall .',
 			runCmd: '',
 			stopCmd: '',
-			debugCmd: 'python -m flask run --host=0.0.0.0 --port=5858 --debugger',
+			debugCmd: 'echo No debug command specified in cli-config',
```
Comment (Aerex) [inline generators/dockertools/index.js]: Same as previous message

File: generators/dockertools/templates/python/Dockerfile-tools [hunk 3]
```diff
@@ -4,10 +4,10 @@ WORKDIR /app
 
 COPY requirements.txt /tmp/requirements.txt
 RUN apk --update add python py-pip openssl ca-certificates py-openssl wget linux-headers
-RUN apk --update add --virtual build-dependencies libffi-dev openssl-dev python-dev py-pip build-base \
-  && pip install --upgrade pip \
-  && pip install -r /tmp/requirements.txt \
-  && apk del build-dependencies
+RUN apk --update add --virtual build-dependencies libffi-dev openssl-dev python-dev py-pip build-base
```
Comment (Aerex) [inline generators/dockertools/templates/python/Dockerfile-tools]: Revert this change

File: generators/dockertools/templates/python/Dockerfile-tools [hunk 4]
```diff
@@ -4,10 +4,10 @@ WORKDIR /app
...
+RUN pip install --upgrade pip
+RUN pip install -r /tmp/requirements.txt
```
Comment (Aerex) [inline generators/dockertools/templates/python/Dockerfile-tools]: Revert this change

**Comment by Aerex** [comment]
Issue blocked by arf/planning-node#222

File: generators/dockertools/index.js (hunk 1 above)
Comment (gabe-terrell) [reply generators/dockertools/index.js]: Okay, I changed back `testCmd` and `debugCmd`, but what are your thoughts on just keeping the `buildCmd`s as `python -m compileall .` instead of `python -m compileall server test`? If the user ever adds a new directory for additional python files, the `compileall` will still hit them without concern. Performance wise, this shouldn't really make much of a difference I'd imagine.

Comment (Aerex) [reply generators/dockertools/index.js]: Yeah this is fine.
